public/js/eventList.js

```diff
@@ -0,0 +1,36 @@
+
+document.querySelectorAll(".event .lower").forEach(function(event) {
```

Comment:
'event' isn't very semantic, it's the same name you used for a different element on line 7.

Reply:
they are the same thing, event points at each of the event divs

Reply:
it turns out removing .event makes no change, so I removed it

97fbeed